### PR TITLE
Bump zk version to 3.4.11 and add line to install service

### PR DESCRIPTION
**Pull request by bndabbs**
3.4.9 doesn't seem to exist anymore

There was no line to install the service file, so it wasn't getting
written

### CODE REVIEW

**Comment by dcode** [comment]
@bndabbs looks good. Can you add a changelog section with your changes. This one missed my quality checks when I originally built it.

File: zookeeper/zookeeper.spec [hunk 1]
```diff
@@ -46,6 +46,7 @@ install -p -m 0644 zookeeper-%{version}.jar lib/*.jar \
   %{buildroot}%{zk_prefix}/
 # Service, systemd fails to expand file paths in runtime
 mkdir -p %{buildroot}%{_unitdir}
+install -p -D -m 0644 %{S:1} %{buildroot}%{_unitdir}/zookeeper.service
 CLASSPATH=
 for i in %{buildroot}%{zk_prefix}/*.jar; do
   CLASSPATH="%{zk_prefix}/$(basename ${i}):${CLASSPATH}"
```
Comment (dcode) [inline zookeeper/zookeeper.spec:52]: Please add a changelog section with your specific changes.

**Review by dcode — CHANGES_REQUESTED** [review]

**Comment by bndabbs** [comment]
Ah, that makes sense. Updating...

**Review by dcode — APPROVED** [review]
LGTM

**Pull request merged**